### PR TITLE
Milestone2

### PR DESCRIPTION
**Pull request by epinczinger**
On this branch, I've worked on the following specifics:

- Create App, Display, Button and ButtonPanel components.
- Add default props and prop types to Button and Display components.
- The app is already rendered.
- Eslint errors have been fixed.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+export default function App() {
+  return (
+    <div>
+      <Display />
+      <ButtonPanel />
+    </div>
```
Comment (wathigo) [inline src/components/App.js]: **REQUIRED**
* Has children components wrapped in React.Fragment short syntax ([screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png))

**Review by wathigo — CHANGES_REQUESTED** [review]
## Changes Required :o: 
Great job on this milestone.
Let's make some changes before we move on. See the comment :point_down: 

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

**Comment by epinczinger** [comment]
The requested change has been done. Submitting for another code review.

-Added the shorthand of React.Fragment to avoid extra node.

**Review by teekaytech — APPROVED** [review]
## Status: Approved :green_circle: 

### Congratulations @epinczinger :tada: 

**Thank you for implementing the changes required by the last reviewer :handshake: 
This milestone has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:**

**Comment(s)**
- Against your next milestone, kindly fix the broken `Issues page` and `MIT license` links in your Readme.


Again, congratulations, you can now merge.
Good luck with your next task.

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**